enhancements/audit-command.md

```diff
@@ -0,0 +1,722 @@
+---
+title: audit-command-operation
+authors:
+  - "@camilamacedo86"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-02-22
+last-updated: 2021-03-02
+status: implementable
+see-also:
+  - [catalog builds in operator-sdk](operator-sdk-builds-catalogs.md)
+  - [custom operator validation #43](https://github.com/operator-framework/enhancements/pull/43)
+  - [Enhancement for validating bundles in a catsrc image #48](https://github.com/operator-framework/enhancements/pull/48)
+  - [operator-framework/api: add new checks to the operatorhub validator][ep-new-checks for-operator-hub]
+---
+
+# Audit command operation
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions 
+
+1. Should this command to be added to SDK or OPM or in a new tool?
+
+**Context**
+
+The audit command will perform checks that are implemented and are provided via SDK tool commands `bundle validate` and `scorecard` already. Then, it could be done from SDK binary or use it. However, the OPM tool is the tool that has been used with the purpose of admin the catalogs. 
+
+Note that SDK will also get starts to allow users to work with catalogs by using opm. See the proposal [catalog builds in operator-sdk](operator-sdk-builds-catalogs.md).
+
+Also, the motivations of the proposal [Custom operator validation](https://github.com/operator-framework/enhancements/pull/43)  confirms the interest of OPM users in the audit command. 
+
+The downside of create a new tool for this purpose shows mainly be the effort to keep it maintained and not take advantage of is  implemented already as ask for users which who requires to audit the OLM index catalogs and would like to use this feature also install an additional tool since they probably work already with opm or sdk and maybe both indeed. 
```

Comment:
I don't think it is necessary to have a new tool, I think `opm` makes more logical sense since it already manages the catalogs.

Reply:
Maybe. However, it shows more of a product owner decision. See the open question info and note that your point is addressed there as well. This proposal has the goal to discuss what/how to audit command will work and then, where it will be implemented is a secondary question.